src/main/kotlin/io/zerobase/smarttracing/Main.kt

```diff
@@ -18,7 +19,7 @@ import javax.servlet.FilterRegistration
 
 typealias MultiMap<K,V> = Map<K, List<V>>
 
-data class Config(val database: Neo4jConfig, val siteTypeCategories: MultiMap<String, String>): Configuration()
+data class Config(val database: Neo4jConfig, val siteTypeCategories: MultiMap<String, String>, val scannable: List<String>): Configuration()
```

Comment:
rename the new field to `scannableTypes`. a) the field is a list so the name should be plural and b) it's not a scannable or a list of scannables, it's a list of the valid types and the name should reflect that